vortex/gui.py

```diff
@@ -134,6 +144,7 @@ def evaluate_block(self):
         if code:
             _logger.info(f"Eval: '{code}'")
             try:
+                code = code.replace('\r', '').replace('\n', '')
```

Comment:
Hm, why remove carriage returns and line feeds here?

Reply:
With providing the new stream assignment method instead of calling p(..., ...), the assigned pattern is not a function parameter anymore, it's now the right hand expression of the assingment. If one would like to pass a pattern in a multiline manner, the line continuation character has to be appended to each line of the statement.

Without this change the equivalent statement of:
```
p("test", s(stack("gabba*4", "cp*3").every(3, fast(2)))
    >> speed("2 3")
    >> room(0.5)
    >> size(0.8)
)
```
would look like this:
```
d["test"] = s(stack("gabba*4", "cp*3").every(3, fast(2))) \
    .speed("2 3") \
    .room(0.5) \
    .size(0.8)
```
With this change, the line breaks are removed before evaluating and the backslash is not required in the pattern and the example would look like:
```
d["test"] = s(stack("gabba*4", "cp*3").every(3, fast(2)))
    .speed("2 3")
    .room(0.5)
    .size(0.8)
```

Reply:
Ah I see, thanks.. I didn't know method calls couldn't run across multiple lines. but then I guess this would break patterns that rely on line breaks, e.g. have multiline statements in a lambda?

Reply:
Yes, you are right. I will remove this change an for multiline patterns you can use the backslash or put the whole pattern in brackets:
```
d["test"] = (s(stack("gabba*4", "cp*3").every(3, fast(2)))
    .speed("2 3")
    .room(0.5)
    .size(0.8))
```

Reply:
And with using a diferent UI like JupytherLab, this issue doesn't count anymore.

Reply:
I've updated the branch.

Reply:
Super thanks. We can explore alternatives in the future.